android-kit-base/src/main/java/com/mparticle/kits/MPSideloadedFilters.kt

```diff
@@ -0,0 +1,90 @@
+package com.mparticle.kits
```

Comment:
If this file isn't needed, we should remove it from the PR or just include an empty class definition if the type is used.

This class has things with comments like "web only" or "iOS only" that should not be merged into development/main.

Reply:
The class is needed as the new MParticleSideloaded abstraction has a fuction to addFilters(filters : MPSideloadedFilters) required on tje Jira ticket.

We can delete the comments, but thats all.

Reply:
Actually sorry I should have been clearer in the Jira ticket. It means to add the MPSideloaded kit class only as far as it needs to be implemented to not have to change the public API later. None of the addFilter methods actually work yet, so they shouldn't be there. See the example in the iOS code base, it only has the kit reference and nothing else: https://github.com/mParticle/mparticle-apple-sdk/blob/main/mParticle-Apple-SDK/Kits/MPSideloadedKit.swift

Reply:
OK Ill do that.